.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
       - id: pyupgrade
```

Comment:
run `pre-commit autoupdate` to update all hooks, not just black?

Reply:
👍 Thanks.